lib/adapters/datatip-adapter.ts

```diff
@@ -88,5 +88,8 @@ export default class DatatipAdapter {
         value: markedString.value,
       };
     }
+
+    // Catch-all case
+    return { type: 'markdown', value: markedString.toString() };
```

Comment:
TypeScript compiler wouldn't let this method live without a catch-all case.  Is there a better result value?